Automattic/wp-calypso
# Hosting Configuration: Remove no longer used Clear Cache card CSS

It's follow-up for https://github.com/Automattic/wp-calypso/pull/73360 

## Proposed Changes

In the related PR, we've simplified the Clear Cache card by removing the confirmation dialog and enabling it for all users. In this PR, I propose to remove leftover CSS code which is no longer used.

## Testing Instructions



1. Navigate to Atomic site, Settings -> Hosting Configuration
2. Locate the Cache block at the bottom of the page
3. Confirm that Clear cache block still looks fine

## Pre-merge Checklist
